wordpress-mobile/WordPress-Android
# Issue/4180 menu api handle error

Fixes #4180 

To test: 

1) go ahead to the Menus section 
2) start creating a new menu (by clicking on `+ Add new menu ` in the menus dropdown) 
3) enter the exact same name of an existing menu (respecting upper/lower-case) as any of the ones displayed in the dropdown used in step 2 above. 
4) tap on the SAVE button (make sure you've got connectivity at this point)
5) a toast should appear showing the error, and it should be localized. 

For this, try doing it in English and then switching the device language setting to other such as Spanish or Deutsch (or whatever other language you feel comfortable enough with to navigate 😄 ), and repeat steps 1-5 above.


Needs review: @nbradbury @tonyr59h
